packages/create-vite/template-lit-ts/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "module": "esnext",
-    "lib": ["es2017", "dom", "dom.iterable"],
+    "module": "ESNext",
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
```

Comment:
Just to note: I replaced all `module`, `moduleResolution`, `lib` and `target` fields with there auto-suggested non lower case values. (You get that when using ctrl+space)